### PR TITLE
adding trigrams

### PR DESCRIPTION
**Pull request by jayjonesseattle**
hi, I looked at my progress and realized I didn't complete this problem. didn't want it to bring down my grade. honestly, it is a bit confusing for me.  now quite understanding the directions for the final piece I guess.

### CODE REVIEW

File: students/terrance_jones/lesson4/trigrams.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import string
+import random
+
+file = 'sherlock_short.txt'
+
+
+def read_file():
+    with open(file, 'r') as f:
+        text = f.read()
+
```
Comment (charlesrobison) [inline students/terrance_jones/lesson4/trigrams.py]: You can remove the extra blank lines in these functions.

File: students/terrance_jones/lesson4/trigrams.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        #creates a string for dic key from the pair list
+        string = pair[0] + " " + pair[1]
+
+        if string in trigram_dic.keys():
+            trigram_dic[string].append(follower)
+        else:
+            trigram_dic[string] = [follower]
+
+    
+    return(trigram_dic)
```
Comment (charlesrobison) [inline students/terrance_jones/lesson4/trigrams.py]: This function doesn't work for me - getting NameError: name 'words' is not defined

File: students/terrance_jones/lesson4/trigrams.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    
+
+
+
+    if __name__ == '__main__':
+    trigrams = build_trigrams(words)
```
Comment (charlesrobison) [inline students/terrance_jones/lesson4/trigrams.py]: Indention errors here.

**Comment by natasha-aleksandrova** [comment]
@jayjonesseattle we'll merge what you have so far, please do a new PR and please reference this PR in the new submission so we have context. thank you!

**Pull request merged**